secret/Tiltfile

```diff
@@ -1,13 +1,15 @@
 # -*- mode: Python -*-
 
-def secret_yaml_generic(name, namespace="", from_file=None, secret_type=None):
+def secret_yaml_generic(name, namespace="", from_file=None, from_env_file=None, secret_type=None):
```

Comment:
i would probably copy the kubectl documentation:

```
--from-env-file='': Specify the path to a file to read lines of key=val
pairs to create a secret (i.e. a Docker .env file)
```

and let kubectl handle the error messaging. I think it will do the right thing:

```
$ kubectl create secret generic my-secret --from-file=path/to/bar --from-env-file=path/to/bar
error: from-env-file cannot be combined with from-file or from-literal
```